### PR TITLE
Translate some new English content

### PR DESCRIPTION
**Pull request by Jinjiang**
## Description of Problem

- `api/application-api.md`
- `api/built-in-components.md`
- `api/composition-api.md`
- `api/global-api.md`
- `guide/installation`
- `guide/transitions-list`
- `guide/ssr`
- `guide/migration/async-components`
- `guide/migration/custom-directives`
- `guide/migration/emits-option`
- `guide/migration/transition-as-root`
- `guide/migration/transition-group`
- `guide/migration/transision`
- `cookbook/automatic-global-registration-of-base-components.md`

### CODE REVIEW

File: src/api/built-in-components.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
 # 内置组件
 
-<!-- TODO: translation -->
-Built-in components can be used directly in templates without needing to be registered.
+内置组件可以直接在模板中使用，而不需注册。
 
-The `<keep-alive>`, `<transition>`, `<transition-group>`, and `<teleport>` components can all be tree-shaken by bundlers, so that they are only included in the build if they're used. They can also be imported explicitly if you need direct access to the component itself:
+`<keep-alive>`、`<transition>`、`<transition-group>` 和 `<teleport>` 组件都可以被打包工具 tree-shake。所以它们只会在被使用的时候被引入。如果你需要直接访问它们，也可以将它们显性导入。
```
Comment (veaba) [inline src/api/built-in-components.md:5]:
```suggestion
`<keep-alive>`、`<transition>`、`<transition-group>` 和 `<teleport>` 组件都可以被打包工具 tree-shaking。所以它们只会在被使用的时候被引入。如果你需要直接访问它们，也可以将它们显性导入。
```

Comment (veaba) [reply src/api/built-in-components.md:5]: 原文是不是写错了，`tree-shaken` 一般写为 `tree-shaking`

File: src/api/built-in-components.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
 ```
 
 ```js
-// ESM build of Vue
+// Vue 的 ESM 构建
 import { KeepAlive, Teleport, Transition, TransitionGroup } from 'vue'
 ```
 
-`<component>` and `<slot>` are component-like features of template syntax. They are not true components and they can't be imported like the components shown above.
+`<component>` 和 `<slot>` 是模板语法中组件形式的特性。它们不是真正的组件且无法像上述组件意义被导入。
```
Comment (veaba) [inline src/api/built-in-components.md:17]:
```suggestion
`<component>` 和 `<slot>` 是模板语法中组件形式的特性。它们不是真正的组件且无法像上述组件那样被导入。
```

File: src/api/global-api.md [hunk 3]
```diff
@@ -525,10 +525,9 @@ export default {
 
   CSS 模块的名称。默认为 `'$style'`。
 
-<!-- TODO: translation -->
 ## version
 
-Provides the installed version of Vue as a string.
+以字符串形式提供被安装的 Vue 的版本号。
```
Comment (veaba) [inline src/api/global-api.md:530]:
```suggestion
以字符串形式提供已安装的 Vue 的版本号。
```

File: src/cookbook/automatic-global-registration-of-base-components.md [hunk 4]
```diff
@@ -1,11 +1,10 @@
...
 
-<!-- TODO: translation -->
-## Base Example
+## 基础示例
 
-Many of your components will be relatively generic, possibly only wrapping an element like an input or a button. We sometimes refer to these as [base components](../style-guide/#base-component-names-strongly-recommended) and they tend to be used very frequently across your components.
+许多组件都是相对一般化的，可能只是包裹了一个输入框或按钮之类的元素。我们有时将其称为[基础组件](../style-guide/#基础组件名称强烈推荐)并且它们通常会在不同的组件中频繁被使用。
 
-The result is that many components may include long lists of base components:
+结果就是很多组件都引入了一个基础组件都长列表：
```
Comment (veaba) [inline src/cookbook/automatic-global-registration-of-base-components.md:7]:
```suggestion
结果就是很多组件都引入了一个基础组件的长列表：
```

File: src/api/built-in-components.md (hunk 1 above)
Comment (Justineo) [reply src/api/built-in-components.md:5]: 同另一个 PR，建议不要加时态。

Comment (Justineo) [reply src/api/built-in-components.md:5]: tree-shake 是动词，tree-shaken 是完成时，tree-shaking 是进行时当名词用。

File: src/guide/transitions-list.md [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 那么怎么同时渲染整个列表，比如使用 `v-for`？在这种场景中，使用 `<transition-group>` 组件。在我们深入例子之前，先了解关于这个组件的几个特点：
 
-- By default, it doesn't render a wrapper element, but you can specify an element to be rendered with the `tag` attribute. <!-- TODO: translation -->
+- 默认情况下，它不会渲染一个元素的包裹器，但是你可以制定一个元素并以 `tag` attribute 进行渲染。
```
Comment (veaba) [inline src/guide/transitions-list.md:10]:
```suggestion
- 默认情况下，它不会渲染一个元素的包裹器，但是你可以指定一个元素并以 `tag` attribute 进行渲染。
```

**Review by veaba — APPROVED** [review]

**Pull request merged**